.github/workflows/check-with-osv-detector.yml

```diff
@@ -0,0 +1,20 @@
+name: Check dependencies for known security vulnerabilities
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  check-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check dependencies for security vulnerabilities
+        uses: g-rath/check-with-osv-detector@main
+        # update this if your project has lockfiles in nested directories
+#        with:
+#          # prettier-ignore
+#          paths:
+#            backend/
+#            frontend/
```

Comment:
This should probably be rolled into #356 when it lands rather than being a whole separate workflow eh

Reply:
yeah that is probably the way to go - we don't even need a dedicated variant, just need to axe the current `audit-app` one